src/renderer/styles/helpers.js

```diff
@@ -22,6 +22,11 @@ export const lighten = (c: string, a: number) =>
     .lighten(a)
     .toString();
 
+export const mix = (c: string, b: string, a: number) =>
+  Color(c)
+    .mix(Color(b), a)
+    .toString();
```

Comment:
I have a feeling that this might not be super performant, but let's see.

Reply:
this was needed in order to get the right shade without transparency in order to stack operation icons